charts/matrix-server/values.yaml

```diff
@@ -5,8 +5,8 @@ port: 8008
 domain: web3.foundation
 
 image:
-  repo: matrixdotorg/synapse
-  tag: v0.99.5.2
+  repo: web3f/synapse
```

Comment:
I think we should not change it here, instead we should set the antiscam values in the community matrix server, here https://github.com/w3f/community-matrix-server/blob/master/helmfile.d/10-matrix.yaml#L13 This way other matrix servers will be unaffected.